# origin: bjorn.helgaas (BitKeeper)
# cset: 1.1461 (2.4) key=42336f56oUNh6k9KKMFrbmGgsap8yQ
# URL: http://linux.bkbits.net:8080/linux-2.4/cset@42336f56oUNh6k9KKMFrbmGgsap8yQ
# inclusion: upstream
# descrition: ia64: add missing syscall-slot.
# revision date: Wed, 11 May 2005 18:18:46 +0900
#
# S rset: ChangeSet|1.1460..1.1461
# I rset: arch/ia64/kernel/entry.S|1.15..1.16
#
# Key:
# S: Skipped  ChangeSet file only
# O: Original Followed by Updated
# U: Updated  Included with updated range of versions
# I: Included Included verbatim
# E: Excluded Excluded on request from user
# D: Deleted  Manually deleted by subsequent user edit
# R: Revised  Manually revised by subsequent user edit
#
#
# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/03/12 15:38:14-07:00 bjorn.helgaas@hp.com 
#   ia64: add missing syscall-slot.
# 
# arch/ia64/kernel/entry.S
#   2005/03/12 15:27:20-07:00 bjorn.helgaas@hp.com +1 -0
#   Add missing syscall-slot.
# 
#
===== arch/ia64/kernel/entry.S 1.15 vs 1.16 =====
--- 1.15/arch/ia64/kernel/entry.S	2003-07-30 22:33:09 +09:00
+++ 1.16/arch/ia64/kernel/entry.S	2005-03-13 07:27:20 +09:00
@@ -1443,3 +1443,4 @@ sys_call_table:
 	data8 ia64_ni_syscall
 	data8 ia64_ni_syscall
 	data8 ia64_ni_syscall
+	data8 ia64_ni_syscall
